organization/_posts/2020/2020-04-23-coordination.md

```diff
@@ -0,0 +1,210 @@
+
+### Reconstruction and Software Triggers
+
+- DL: There are some overlapping ideas to consider consolidate with respect to the facilities document to LHCC (perhaps more general than the reco/trigger secion). 
+- One idea we didn't capture so well is the need to support HPC architectures. This comes out in both validation and non-x86 support. Will add a few sentences today.
+ 
+### Data Analysis
+
+- Improved the coherence of the document since last  week, we added several important points for ~the same word count but could maybe be more concise.  Some references need to be found/fixed.
+- We could still be a little more forthright in the R&D needs section and point out what we think useful directions are. For example, pointing out how beneficial it has been to open up to new ways of thinking (DataFrames et al), this healthy competition philosophy should continue.
+
+### ~~Conclusions~~ Summary
+
+- *Summary* is a more approriate way to finish the document
+- Bullet points per section (aiming for 4-5 seems about right)
+    - 24 month timescale (stated as 'a few years')
+    - Should be realistic, but ambitious
+- Final conclusion points
+    - We need investment in software
+    - Adaption to accelerators (particularly GPUs) is a common theme
+    - Stakeholder input is key
+    - HSF has a coordination role
+    - Mention IRIS-HEP, SIDIS, other existing projects?
+        - Could at minimum have a link to last year's JLab workshop session / European Strategy submission 
+        - Prefer to mention things we know are funded
+    - Mention links with ESCAPE? (currently mentioned in trigger&reco chapter)
+        - As long as relevant to the HL-LHC focus
+
+---
+
+## Working Group Updates
+
+### Data Analysis
+
+Nothing to report, occupied with the LHCC report.
+
+### Detector Simulation
+
+For the last couple of weeks focussed on the LHCC report. Need to organize the next topical meeting asap avoiding conflicts with May workshop.
+
+### Reconstruction and Software Triggers
+
+#### Meetings
+
+No meeting in May due to workshop, but some (incl. categories) potentially lined up after interactions for LHCC doc: 
+
+- Accelerators in trigger & reconstruction (Allen/Patatrack updates since CHEP + contribution from ATLAS, topics can be chosen following HSF workshop contributions)
+- [category of meetings] How experiments sharing pieces of LHC experiments can reuse LHC trigger & reco code. Cases in mind: FASER, LDMX (but maybe more, e.g. future colliders?)
+- CBM experiment and 4D (time-slice) tracking reconstruction
+- FASTML (HSF4ML & SONIC) for ML on accelerators
+- (from earlier) Particle flow review
+
+Would it be worth categorizing meetings according to priorities in LHCC document? May benefit HSF PR...
+
+*Using the LHCC document to plan the next activities for the working groups is strongly supported.*
+
+#### Other
+
+We have a request of someone who wanted to work in trigger & reco (see previous meetings). 
```

Comment:
We have a request _of_ someone who _wanted_ to work _in_ => We have a request **from** someone who **wants** to work **on**?